src/subword.rs

```diff
@@ -157,13 +157,85 @@ impl SubwordIndices for str {
     }
 }
 
+/// Create an Iterator going through a sentence
```

Comment:
sentence -> token

Maybe something like:

```
Iterator over the n-grams in a word and the corresponding subword indices.

`NGramsIndicesIter` is an iterator that produces the n-grams in a word and
the corresponding subword indices as tuples `(ngram, index)`.
```

I wouldn't mention the types, since they are already added automatically to the rustdoc.